applications/forms.py

```diff
@@ -137,6 +137,12 @@ def clean_other_gender(self):
             raise forms.ValidationError("Please enter this field or select 'Prefer not to answer'")
         return data
 
+    def clean_origin(self):
+        data = self.cleaned_data['origin']
+        if data.count(',') != 2:
+            data = "Others"
+        return data
```

Comment:
How about we raise an issue to the user instead of silently changing this? Also please change help_text to make sure we are calling this out: `Select one of the dropdown options`

Also, this `,` clean up seems rather arbitrary, wondering if this check can be improved.

Reply:
Maybe warn at first like If you use one of the options your option will be Others, and the if s/he submits again the option will be Others?

Reply:
I would fail always. But make the test a bit more reliable (maybe use the autocomplete API here to check the name is valid)